### ISSUE TITLE
[display] "No classes found in <enum value>" with position completion

### ISSUE THREAD

**Issue by Gama11** [HaxeFoundation/haxe#5119]
```haxe
class Main { public static function main() { foo(A); }
	static function foo(key:Key) {}
}
```

Must be in a separate `Key.hx` file:

```haxe
@:enum
abstract Key(String) {
	var A = "A";
}
```

```
haxe --display Main.hx@49@position
```

```
No classes found in A
```

Works as expected when passing the fully qualified name (`foo(Key.A)`) or when `Key` is in the same file.

**Comment by Simn** [comment]
```
Client connected
Reading 259 bytes
Processing Arguments [--cwd,c:\Users\simn\Documents\TestMisc,build.hxml,-D,display-details,--no-output,--display,c:/Users/simn/Documents/TestMisc/src/Main.hx@49@position,-D,display-stdin]
Defines dce=std,display-details=1,display-stdin=1,display=position,display_details=1,display_stdin=1,haxe3=1,haxe_ver=3.3,jquery_ver=11203,js-es5=1,js=1,js_es5=1,js_es=5,no_copt=1,source-header=Generated by Haxe 3.3.0,true=1
Using signature 671ab7bd39d8099e19d258eb1ddc99a4
Completion Response =
<list>
<pos>c:\users\simn\documents\testmisc\src\key.hx:3: characters 4-16</pos>
</list>
Stats = 2 files, 8 classes, 2 methods, 0 macros
Time spent : 0.007s
```

That seems to work for me.

**Comment by Gama11** [comment]
Interesting. I get the same result as you on latest dev (308f25a), but with de2342c which I used yesterday, I can still reproduce the original error.

Apart from that, is `key.hx:3: characters 4-16` really the expected result here? I'd expect the position of `foo()` is the result here (`main.hx:2: characters 11-35`). That's actually what I get with `@48`.

On 3.2.1, I get the `main.hx` result with `@49` as well (but an empty list with `@48`).

**Comment by Gama11** [comment]
Btw, is there a reason why the result's character range doesn't include the `static` modifier of `foo()`?

**Comment by Gama11** [comment]
This can be closed with the conclusion that 3.2.1 was broken regarding the positions passed to position completion.
